src/screens/ChatDetailPage/index.js

```diff
@@ -15,7 +15,7 @@ import {COLORS} from '../../utils/theme';
 import {Context} from '../../context';
 import {CustomMessageSystem} from '../../components';
 import {fonts} from '../../utils/fonts';
-import {setAsset, setParticipants} from '../../context/actions/groupChat';
+import {setAsset} from '../../context/actions/groupChat';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setParticipants` function is removed from the import statement but it seems to be used in the old code. If it's not replaced or its usage is not removed, this could lead to a reference error.
```diff
- import {setAsset, setParticipants} from '../../context/actions/groupChat';
+ import {setAsset} from '../../context/actions/groupChat';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->